.gitignore

```diff
@@ -92,6 +92,7 @@ unlinked_spec.ds
 **/ios/Flutter/Flutter.framework
 **/ios/Flutter/Flutter.podspec
 **/ios/Flutter/Generated.xcconfig
+**/ios/Flutter/ephemeral
```

Comment:
The other non-template `.gitignore`s aren't necessary on top of this one, but let's be consistent.